test-packages/custom-sandbox-app/README.md

```diff
@@ -0,0 +1,58 @@
+# custom-sandbox-app
```

Comment:
@xg-wang what is the difference between this test app and the basic-app?

If there is a difference and you update the read me there to be clear about what this test-package app is doing and what tests should be added to this package. 

You could use this as an example
https://github.com/ember-fastboot/ember-cli-fastboot/tree/master/test-packages

Reply:
Pushed a commit to update the readme. The difference to `basic-app` is minimal, but we still need a separate app to build. Any suggestions?